web_advanced_search_x2x/static/src/js/web_advanced_search_x2x.js

```diff
@@ -180,7 +187,7 @@ openerp.web_advanced_search_x2x = function(instance)
         popup_domain_selection: function()
         {
             var self = this,
-                popup = new instance.web_advanced_search_x2x.SelectCreatePopup(this);
+                popup = new form_common.SelectCreatePopup(this);
```

Comment:
I think you lose the "use criteria" function this way, which is one of the most powerful features of this module imho